_posts/2021-2-01-Workday-Overview.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: Website Overview
+description: 
+summary: 
+tags: 
+---
+Today, I started work with Typing Club and made some progress. After Typing Club, I continue adding to my website. This time I added a search menu to my website and added functional links for the menu to work. I added the input on the main html page, and added the styling and functioning to style.css and script.js. The Css file is meant to stylize the layout of the menu, and you can change the length and width of the search bar to your liking. The script file is meant to activate the search menu since the functionality of the menu is focused on the script of the code. For the search menu to work, you add the html file name in the `<ul>` code along with `<li>` within the `<ul>` code. This allos for the search to take you the destination of the site with the content. [Here is how my website works](https://osvaldo178.github.io/Thinkment/)I than created a pull request for Kajari to look over and leave comments on the changes to commit on the coding.
```

Comment:
/Thinkment/)I than   --> leave a space after the `)` and `than` should be `then`